R/xgb.R

```diff
@@ -93,21 +76,3 @@ axe_env.xgb.Booster <- function(x, verbose = FALSE, ...) {
     verbose = verbose
   )
 }
-
```

Comment:
I think we should probably remove `axe_env.xgb.Booster` too. It seems like callbacks are an optional feature, and it _seems_ like they are only used at training time, but I can't confirm this. Since they are optional and probably don't take up space by default, I think removing this potentially harmful method is the safer thing to do

Reply:
Hmm, I see 

```
#>  1 callbacks.cb.evaluation.log   0.0354  
#>  2 callbacks.cb.print.evaluation 0.0146  
```

in the weigh results, which does get smaller when you axe it.

I guess those are default callbacks that get added by xgboost. I guess we should leave this method in there then